src/performance_counters/register_counters.cpp

```diff
@@ -41,7 +43,14 @@ namespace phylanx { namespace performance_counters
             value.status_ = hpx::performance_counters::status_new_data;
```

Comment:
You need to call `reinit` here if it has never been called before.

Reply:
Please check again.